spring-cloud-kubernetes-integration-tests/run.sh

```diff
@@ -142,15 +93,20 @@ main() {
 
 	cd $CURRENT_DIR
 
-    # TODO: invoke your tests here
-    cd spring-cloud-kubernetes-core-k8s-client-it
-    ../../mvnw clean install -P it
-    cd ../
-     cd spring-cloud-kubernetes-client-config-it
-    ../../mvnw clean install -P it
-    cd ../
-    cd spring-cloud-kubernetes-configuration-watcher-it
-    ../../mvnw clean install -P it
+	WATCHER_IMAGE="docker.io/springcloud/spring-cloud-kubernetes-configuration-watcher:${MVN_VERSION}"
```

Comment:
Maybe we have a way of loading additional images for certain tests?

Reply:
declared the image names as an array variables

Reply:
sounds good